### PR TITLE
feat: bottom tab

### PR DESCRIPTION
**Pull request by Mokto**

### CODE REVIEW

File: src/modules/authenticated/authenticated.tsx [hunk 1]
```diff
@@ -1,43 +1,110 @@
...
+              title: screen.title,
+              tabBarIcon: ({ color, focused }) => (
+                <Icon
+                  name={focused ? screen.iconActive : screen.icon}
+                  color={color}
+                  size={26}
+                />
+              ),
+              // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+              // @ts-ignore
```
Comment (mfateminia) [inline src/modules/authenticated/authenticated.tsx]: no other way?

File: src/modules/authenticated/sponsors/index.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React, { memo } from 'react';
+
+import { StyleSheet, View } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+import { Text } from '../../../shared/text';
+import { Button } from '../../../shared/button';
```
Comment (mfateminia) [inline src/modules/authenticated/sponsors/index.tsx]: can't we use absolute path?

File: src/modules/authenticated/authenticated.tsx (hunk 1 above)
Comment (Mokto) [reply src/modules/authenticated/authenticated.tsx]: No :(

File: src/modules/authenticated/sponsors/index.tsx (hunk 2 above)
Comment (Mokto) [reply src/modules/authenticated/sponsors/index.tsx]: Yes, that's planned for later ;)

**Pull request merged**